wandb/apis/public/reports.py

```diff
@@ -106,16 +119,23 @@ def __repr__(self):
 
 
 class BetaReport(Attrs):
-    """BetaReport is a class associated with reports created in wandb.
+    """Represents a report object in the W&B system.
 
-    WARNING: this API will likely change in a future release
+    Provides access to report attributes (name, description, user, spec,
```

Comment:
I would change `name` to `display_name` that is the name the user would be familiar with. `name` might be deprecated later but for now want to keep this backwards compatible.

Reply:
yes that language would be great.